### PR TITLE
feature/Add-Mortgage

### PR DESCRIPTION
**Pull request by nishatoma**
Added Mortgage base files. Will add more.

### CODE REVIEW

File: app/src/main/java/com/td/mylevelup/mortgage/mortgageCalculatorPage/MortgageActivity.kt [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+        setUpSpinner()
+
+        button.setOnClickListener { _ ->
+
+            if (isValidForm()) {
+                val principal: String = principalAmountEditText.value!!
+                val interestRate: String = interestRateEditText.value!!
+                val amortization: String = amortizationEditText.value!!
+
+//                val monthlyPayments = CalculatorUtils.calculatePayment(principal,
```
Comment (TheRishabhB) [inline app/src/main/java/com/td/mylevelup/mortgage/mortgageCalculatorPage/MortgageActivity.kt:99]: If you're not using this we can remove it

File: app/src/main/java/com/td/mylevelup/mortgage/mortgageCalculatorPage/MortgageActivity.kt [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+        elements.add(frequencyLabel)
+
+        formBuilder.addFormElements(elements)
+
+        setUpSpinner()
+
+        button.setOnClickListener { _ ->
+
+            if (isValidForm()) {
+                val principal: String = principalAmountEditText.value!!
```
Comment (TheRishabhB) [inline app/src/main/java/com/td/mylevelup/mortgage/mortgageCalculatorPage/MortgageActivity.kt:95]: Don't use !!, use ?:.

EG: `thing.value ?: ""`
or `thing.value ?: 0`

File: app/src/main/java/com/td/mylevelup/mortgage/mortgageCalculatorPage/MortgagePageAdapter.kt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+package com.td.mylevelup.mortgage.mortgageCalculatorPage
+
+import com.ngam.rvabstractions.general.AbstractAdapter
+
+class MortgagePageAdapter(private val presenter: MortgagePagePresenter): AbstractAdapter() {
+
+    override fun buildRows() {
+        // Not using adapter, using
```
Comment (TheRishabhB) [inline app/src/main/java/com/td/mylevelup/mortgage/mortgageCalculatorPage/MortgagePageAdapter.kt:8]: Incomplete comment

**Review by TheRishabhB — DISMISSED** [review]
Minor things. Other than that we're good. Good job

**Comment by nishatoma** [comment]
Will fix later.

**Pull request merged**